packages/website/docs/who-can-apply.md

```diff
@@ -1,61 +1,41 @@
 ---
 title: Who can apply?
+authors:
+  - name: Clément Vannicatte
+    title: Software Engineer @ Algolia
+    url: https://github.com/shortcuts
 ---
 
-We built DocSearch from the ground up with the idea of improving search on large
-technical documentations. For this reason, we are offering a free hosting
-version to all online technical documentations.
+We built DocSearch from the ground up with the idea of improving search on large technical documentations. For this reason, we are offering a free hosting version to all online technical documentations of open-source projects and technical blogs.
 
-We're always sad to have to turn down applications, but with the number of
-requests we receive every day, we've had to focus on fair criteria. We hope you
-understand.
+We're always sad to have to turn down applications, but with the number of requests we receive every day, we've had to focus on fair criteria. We hope you understand.
 
 ## The checklist
 
-To have your request validated, we'll ask that you check all the following
-points.
+To have your request validated, we'll ask that you check all the following points.
 
-- You must be the **owner** of the website, or at least have the permissions to
-  update its content. You'll have to [include a JavaScript snippet][1] to enable
-  DocSearch.
+- You must be the **owner** of the website, or at least have the permissions to update its content. You'll have to [include a JavaScript snippet][1] to enable DocSearch.
 
-- Your website must be **publicly available**. We do not host search indices for
-  websites that are available behind an authentication or a private network.
+- Your website must be **publicly available**. We do not host search indices for websites that are available behind an authentication or a private network.
 
-- Your website must be a **documentation website**. We do not index blogs or
-  commercial content.
+- Your website must be a **technical documentation of an open source project or a technical blog**. We do not index commercial content.
 
-- Your website must **have some final content**. We won't index empty websites
-  nor those filled with lorem ipsum placeholder content. Please, wait until you
-  have written some documentation before applying. We would be happy to help you
-  as soon as you have a steady design.
+- Your website must **have some final content**. We won't index empty websites nor those filled with lorem ipsum placeholder content. Please, wait until you have written some documentation before applying. We would be happy to help you as soon as you have a steady design.
```

Comment:
Not sure about `final content`, prod ready or something similar might be more clear.

Reply:
Indeed, [commit](https://github.com/algolia/docsearch/pull/1048/commits/e1399bc2f189aa7b4cb3d13bddcf62ff39287c3f#diff-c685d47ff3a9e15566050cf8ace0c621da322863f00fbf1bda7ea79f87eafa68R23)